lib/hpax/table.ex

```diff
@@ -3,18 +3,22 @@ defmodule HPAX.Table do
 
   defstruct [
     :max_table_size,
+    :huffman,
     entries: [],
     size: 0,
     length: 0
   ]
 
   @type t() :: %__MODULE__{
           max_table_size: non_neg_integer(),
+          huffman: huffman_strategy(),
           entries: [{binary(), binary()}],
           size: non_neg_integer(),
           length: non_neg_integer()
         }
 
+  @type huffman_strategy() :: :always | :never
```

Comment:
Is there a reason this is not a boolean?

Reply:
Future proofing, mostly. It's possible (likely, even) that we may want other policies for eg: only Huffman encoding strings over a certain length, or perhaps not encoding certain sensitive headers per RFC7541 7.2. This let's us add such policies without breaking the API.